docs/opc-event-output.md

```diff
@@ -0,0 +1,199 @@
+# Detailing the output for OPC UA Events in OPC Publisher
+
+This section describes what the output looks like when listening for events in the OPC Publisher. Events are produced in the PubSub format specified in the OPC UA Standard. The payload is an event which consists of fields selected in the select clause and its values.
+
+Here is an example of the output we get when listening to events from the Simple Events sample:
+```json
+{
+  "body": [
+    {
+      "MessageId": "43",
+      "MessageType": "ua-data",
+      "PublisherId": "SIMPLE-EVENTS",
+      "DataSetWriterGroup": "SIMPLE-EVENTS",
```

Comment:
The name should be DataSetClassId

Reply:
and a guid

Reply:
its correct here: https://github.com/Azure/Industrial-IoT/blob/master/docs/dev-guides/telemetry-messages-format.md
was it changed to datasetwritergroup?

Reply:
I just ran the current code and generated some output and used that as documentation. We have used merged with main and this is how it looks. Don't know the state of master.